netcore/build-geode-native-netcore.sh

```diff
@@ -0,0 +1,54 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
What is the purpose of this helper script to build? Shouldn't this be tied into the main CMake build process?